recipes/azcopy/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "azcopy" %}
+{% set version = "10.6.1" %}
+{% set release_date = "20201021" %}
+{% set sha256 = "3d4e6359f3df2c237601f2f5dc86bffbfde2356b61c305eb6e5089058f00e1bf" %}  # [osx]
+{% set sha256 = "6982e680943427f360f1ac1e4f70ef05087cf4a07814d4efe600d54705693a83" %}  # [linux]
+{% set sha256 = "8ee0c126d77fb04731cd0b09e7451932dcd8a4c08308a25a8e84ba5db9c08c5a" %}  # [win]
+{% set platform = "linux" %}  # [linux]
+{% set platform = "darwin" %}    # [osx]
+{% set platform = "windows" %}    # [win]
+{% set ext = "tar.gz" %}      # [linux]
+{% set ext = "zip" %}         # [not linux]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://azcopyvnext.azureedge.net/release{{ release_date }}/azcopy_{{ platform }}_amd64_{{ version }}.{{ ext }}
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "cp $SRC_DIR/azcopy $PREFIX/lib/azcopy"  # [not win]
```

I think this should also be `bin`?
```suggestion
  script: "cp $SRC_DIR/azcopy $PREFIX/bin/azcopy"  # [not win]
```

---

I tried that first but it didn't work